easybuild/easyconfigs/l/libev/libev-4.33-GCC-11.2.0.eb

```
@@ -0,0 +1,37 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'libev'
+version = '4.33'
+
+homepage = 'http://software.schmorp.de/pkg/libev.html'
+description = """A full-featured and high-performance (see benchmark) 
+event loop that is loosely modelled after libevent, but without its 
+limitations and bugs. It is used in GNU Virtual Private Ethernet, 
+rxvt-unicode, auditd, the Deliantra MORPG Server and Client, and many 
+other programs."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://dist.schmorp.de/libev']
```

```suggestion
source_urls = ['http://dist.schmorp.de/libev/Attic']
```

It seems like they continuously remove all but the latest versions at this url, so this would quite quickly break. Fortunately, they have a copy already in the Attic

---

Thanks for finding that. I was not aware they are removing older versions. At least they got an Attic where to put things. Lets hope they stay there.